changelog/v1.3.5/add-route-table-weights.yaml

```
@@ -0,0 +1,17 @@
+changelog:
+- type: NEW_FEATURE
+  description: |
```

nit: we want the newlines converted to spaces,
```suggestion
  description: >
```